src/mixin/java/com/gtnewhorizons/angelica/mixins/early/angelica/optimizations/MixinRenderGlobal_ItemRenderDist.java

```diff
@@ -0,0 +1,53 @@
+package com.gtnewhorizons.angelica.mixins.early.angelica.optimizations;
+
+import com.gtnewhorizons.angelica.config.AngelicaConfig;
+import com.llamalad7.mixinextras.sugar.Local;
+import net.minecraft.client.renderer.RenderGlobal;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.item.EntityItem;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(RenderGlobal.class)
+public class MixinRenderGlobal_ItemRenderDist {
+
+    @Unique private static final int[] sodium$entityItemCount = new int[256];
+    @Unique private static int sodium$itemRenderDist = 255;
+
+    @Inject(method = "renderEntities", at = @At(value = "FIELD", target = "Lnet/minecraft/client/renderer/RenderGlobal;countEntitiesRendered:I", ordinal = 0))
+    private void sodium$resetEntitycount(CallbackInfo ci) {
+        int entityCount = 0;
+        boolean reachedLimit = false;
+        final int itemLimit = AngelicaConfig.droppedItemLimit;
```

Comment:
This belongs in SodiumGameOptions & SodiumGameOptionPages as a slider from like 32 to infinity.

The `dynamicItemRenderDistance` mixin application boolean seems... fine as an Angelica Option, but if it's off it should hide the config page.